_static/main-page.css

```diff
@@ -53,7 +53,7 @@ footer {
 }
 
 .dropdown-menu {
-  background-color: #000000d0;
+  background-color: #0d0d0dd0;
```

Comment:
How is this different from `#1a1a1a`?

Reply:
This is 5% grey (darker), the `#1a1a1a` is 10% grey. This gives the dropdown contrast with the background. It's slightly transparent from before, I'm not sure why - just left it the same.

Reply:
Ah, I see. Thanks for the clarification. My only comment is that if you click a dropdown menu buttom (e.g. "Community") and then move the mouse away from the dropdown menu, the dropdown still remains around:

![Screen Shot 2019-11-26 at 4 57 48 PM](https://user-images.githubusercontent.com/11656932/69679571-10a64300-106e-11ea-94ca-76e1e6142a6f.png)

with the extra contrast, it's looks a little off to me having a floating darker menu.

We could try to use same 10% grey (still slightly transparent) for the dropdown background color:

![Screen Shot 2019-11-26 at 4 58 06 PM](https://user-images.githubusercontent.com/11656932/69679585-14d26080-106e-11ea-8d9b-f4c35bea74f0.png)

What do you think about that?

Reply:
I think the contrast helps for the other menu (documentation), otherwise it's not clear it's a dropdown since there's no distinguishable border.

Reply:
I do agree that it looks odd having header not match the dropdown menu. Perhaps there's a way to make the header change color (darken) when selected? We could also go for a lighter grey instead of darker.

Reply:
Good idea. I pushed a commit which will make the navbar darker when hovering over it, or it's been selected. What do you think about that? (Not a problem if we end up reverting that commit)

Reply:
Any thoughts on the new navbar behavior?